### PR TITLE
pinentry: enable pinentry-gnome

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: srcpkgs/pinentry/template [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
  --enable-pinentry-curses --enable-fallback-curses --enable-pinentry-gtk2
- --enable-pinentry-emacs --enable-pinentry-qt --enable-pinentry-tty"
+ --enable-pinentry-emacs --enable-pinentry-qt --enable-pinentry-tty
+ --enable-pinentry-gnome3"
 hostmakedepends="pkg-config"
-makedepends="ncurses-devel gtk+-devel libassuan-devel libgpg-error-devel qt5-devel"
+makedepends="ncurses-devel gcr-devel gtk+-devel libassuan-devel libgpg-error-devel qt5-devel"
 short_desc="PIN or passphrase entry dialogs for GnuPG"
 maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/pinentry/template]: while you're at it please use the SPDX short-identifier

Comment (Cogitri) [reply srcpkgs/pinentry/template]: Like that?

Comment (Cogitri) [reply srcpkgs/pinentry/template]: Did I do it correctly?

**Comment by maxice8** [comment]
please squash everything into one commit

File: srcpkgs/pinentry/template (hunk 1 above)
Comment (maxice8) [reply srcpkgs/pinentry/template]: Yes like that but the license is GPL-2.0-or-later due to the presence of

```
either version 2 of the License, or * (at your option) any later version.  
```

**Pull request merged**